app/events/[id]/page.tsx

```diff
@@ -59,7 +61,16 @@ const EventPage = ({ params }: Props) => {
                 <span className="text-muted-foreground">Venue: </span>
                 {event.venue}
               </h2>
-              <p className="">{event.desc}</p>
+              <div className="prose lg:prose-l">
+                <div className="!text-foreground prose-strong:text-foreground prose-headings:text-foreground">
```

Comment:
instead of changing color for every element, just do `dark:prose-invert` as shown in this article [here](https://tailwindcss.com/blog/tailwindcss-typography-v0-5#easy-dark-mode-support)